src/payment/create-payment-strategy-registry.ts

```diff
@@ -168,7 +170,7 @@ export default function createPaymentStrategyRegistry(
             store,
             orderActionCreator,
             paymentActionCreator,
-            createFormPoster()
+            formPoster
```

Comment:
Maybe also update `SagePayPaymentStrategy` to also use this `FormPoster` instance?